doc/changes/latest.inc

```diff
@@ -155,6 +155,8 @@ Bugs
 
 - :meth:`mne.Report.add_trans` now also works if no digitization points are present in the data (:gh:`10176` by `Jeff Stout`_)
 
+- Argument `verbose` is now respected by dataset fetching (:gh:`10210` by `Mathieu Scheltienne`_)
```

Comment:
Single-backticks for `:py:obj:` (which won't work here I think), double-backticks for code (better here)
```suggestion
- Argument ``verbose`` is now respected by dataset fetching (:gh:`10210` by `Mathieu Scheltienne`_)
```

Reply:
Oops, my bad, thanks!